src/pages/journal/components/test-outcome/test-outcome.scss

```diff
@@ -1,3 +1,15 @@
 test-outcome {
+  .outcome {
 
+  }
+  .start-test-button {
+    background: #00823B;
```

Comment:
Perhaps put these colours in to the colors map?

Reply:
done